src/clndr.js

```diff
@@ -205,10 +211,11 @@
                 if (this.options.lengthOfTime.startDate) {
                     this.intervalStart =
                         moment(this.options.lengthOfTime.startDate)
-                            .startOf('day');
+                            .startOf('week');
```

Comment:
Yea @kylestetz that's correct. And ok, I'll revert it. I didn't know that was something this calendar supported :stuck_out_tongue: I just noticed it when I typed a random date into the test file and it broke what should have been a weekly calendar.